roles/website/tasks/main.yaml

```diff
@@ -0,0 +1,96 @@
+
+- name: Create needed folder structure
+  file: path={{ item }} state=directory recurse=yes owner={{ website_user }} group={{ website_group }} mode=0755
+  with_items:
+  - /home/{{ website_user }}/.local/bin/
+  - /home/{{ website_user }}/.var/jekyll-envs/{{ website_domain }}/
+  - /home/{{ website_user }}/.var/log/
+  - /home/{{ website_user }}/.cache/
+  - /var/www/{{ website_user }}/domains/{{ website_domain }}/
+  - /var/www/{{ website_user }}/cgi-bin/
+
+- name: Clone content repository
+  git: repo=https://github.com/FreifunkBremen/bremen.freifunk.net dest=/home/{{ website_user }}/.var/jekyll-envs/{{ website_domain }}/ force=yes
+  sudo: true
+  sudo_user: "{{ website_user }}"
+
+- name: Install Pygments
+  apt: name=python-pygments
+
+- name: Install Gemfile for Jekyll
+  copy: src=Gemfile dest=/home/{{ website_user }}/.var/jekyll-envs/{{ website_domain }}/ owner={{ website_user }} group={{ website_user }}
+
+- name: Install Jekyll and dependencies via bundler
+  command: bundle install --path .bundle
+  args:
+    chdir: /home/{{ website_user }}/.var/jekyll-envs/{{ website_domain }}/
+  sudo: true
+  sudo_user: "{{ website_user }}"
+
+- name: Install CGI wrapper
+  template: src=php-cgi-starter dest=/var/www/{{ website_user }}/cgi-bin/ mode=755 owner={{ website_user }} group={{ website_user }}
+
+- name: Install .htaccess file
+  copy: src=htaccess dest=/var/www/{{ website_user }}/domains/{{ website_domain }}/.htaccess owner={{ website_user }} group={{ website_user }}
+
+- name: Create log file for Github webhook target
+  file: path=/home/{{ website_user }}/.var/log/github-webhook.log state=touch owner={{ website_user }} group={{ website_user }}
+
+- name: Install Github webhook target
+  template: src=webhook.php dest=/var/www/{{ website_user }}/domains/{{ website_domain }}/ owner={{ website_user }} group={{ website_user }}
+
+- name: Install Jekyll update script
+  template: src=update-jekyll.sh dest=/home/{{ website_user }}/.local/bin/update-jekyll.sh mode=755 owner={{ website_user }} group={{ website_user }}
+
+
+# Meshviewer
+- name: Install ruby-sass
+  apt: name=ruby-sass
+
+- name: Clone Meshviewer repository
+  git: repo=https://github.com/FreifunkBremen/meshviewer.git dest=/home/{{ website_user }}/.cache/meshviewer/ force=yes
+  sudo: true
+  sudo_user: "{{ website_user }}"
+
+- name: Install Meshviewer dependencies
+  npm: path=/home/{{ website_user }}/.cache/meshviewer
+
+- name: Install grunt-cli for Meshviewer
+  npm: name=grunt-cli path=/home/{{ website_user }}/.cache/meshviewer
+
+- name: Build Meshviewer
+  command: chdir=/home/{{ website_user }}/.cache/meshviewer ./node_modules/.bin/grunt
+  args:
+    creates: /home/{{ website_user }}/.cache/meshviewer/build/
+
+- name: Copy built Meshviewer to www directory
+  command: cp -r /home/{{ website_user }}/.cache/meshviewer/build/ /var/www/{{ website_user }}/domains/{{ website_domain }}/meshviewer/
```

Comment:
Wo ist der Vorteil dieser Variante gegenüber einem direkten Bauen in `/var/www` (z.B. via Symlink)? Atomar ist glaube ich keins von beidem.